pkg/scheduler/scheduler_test.go

```diff
@@ -2762,7 +3349,7 @@ func TestResourcesToReserve(t *testing.T) {
 		},
 	}
 	for _, tc := range cases {
-		t.Run(tc.name, func(t *testing.T) {
```

Comment:
It doesn't depend on the gate, so undo this to avoid increase tests' running time. I know it might be negligible, but we might forget about the possibility of optimizing this one if we ever need to optimize.